modules/qr_code.py

```diff
@@ -0,0 +1,59 @@
+import qrcode
+import uuid
+import os
+import logging
+
+from .args import get_args
+
+args = get_args()
+qr = qrcode.QRCode()
+logger = logging.getLogger(__name__)
+
+QRCODE_FOLDER = args.qr_code_folder_path 
+MAX_CACHE_SIZE = args.qr_code_cache_size
+
+class QRCode:
+    def __init__(self) -> None:
+        self.mapping = {}
+
+    def add(self, alias: str):
+        try:     
+            url = f"localhost:8000/find/{alias}" 
```

Comment:
we will need to specify the base path of the url, since locally it will be localhost, and for the real shortner it will be https://sce.sjsu.edu

Reply:
we can define an arg like
```py
parser.add_argument(
        "--qr-code-base-url",
        required=True,
        help="the base url path for the qr code image"
)
```

and then pass that to the constructor of the qr code class, to make the value dynamic. in this case the value would be

```
--qr-code-base-url=http://localhost:8000/find
```

Reply:
resolved from commit [ef544e8](https://github.com/SCE-Development/cleezy/pull/39/commits/ef544e8df418f70231fc02777d96bb462e51ef96)